src/client/Microsoft.Identity.Client/ApiConfig/AbstractManagedIdentityParameterBuilder.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Executors;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.AuthScheme;
+using Microsoft.Identity.Client.Instance;
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Client.TelemetryCore.Internal.Events;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Abstract base class for managed identity application token request builders.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public abstract class AbstractManagedIdentityParameterBuilder<T> : BaseAbstractAcquireTokenParameterBuilder<T>
+        where T : BaseAbstractAcquireTokenParameterBuilder<T>
+    {
+        /// <summary>
+        /// Default constructor for AbstractManagedIdentityParameterBuilder.
+        /// </summary>
+        protected AbstractManagedIdentityParameterBuilder() : base() { }
+
+        internal AbstractManagedIdentityParameterBuilder(IManagedIdentityApplicationExecutor managedIdentityApplicationExecutor) : base(managedIdentityApplicationExecutor.ServiceBundle) 
```

Comment:
nit: add new line before calling base(). It looks a bit cleaner and the line isnt so long.